docs/src/literate/quasi_incompressible_hyperelasticity.jl

```diff
@@ -332,15 +332,13 @@ function solve(interpolation_u, interpolation_p)
         end;
 
         ## Save the solution fields
-        @timeit "export" begin
+        begin
             vtk_grid("hyperelasticity_incomp_mixed_$t.vtu", dh, compress=false) do vtkfile
                 vtk_point_data(vtkfile, dh, w)
                 vtk_save(vtkfile)
                 pvd[t] = vtkfile
             end
         end
```

Comment:
This `begin`/`end` can be removed now:
```suggestion
        vtk_grid("hyperelasticity_incomp_mixed_$t.vtu", dh, compress=false) do vtkfile
            vtk_point_data(vtkfile, dh, w)
            vtk_save(vtkfile)
            pvd[t] = vtkfile
        end
```